### PR TITLE
Notify the user if a project is missing in vh yaml

### PR DESCRIPTION
**Pull request by magdapoppins**
Fixes valohai/valohai-utils#64

### CODE REVIEW

**Review by akx — COMMENTED** [review]
What is the assertion error thrown by valohai_utils? Could we fix the command to work without a project? I see no reason why YAML generation would require a linked project.

**Comment by magdapoppins** [comment]
> What is the assertion error thrown by valohai_utils? Could we fix the command to work without a project? I see no reason why YAML generation would require a linked project.

Requires some decoupling, but should be doable!

File: valohai_cli/commands/yaml/step.py [hunk 1]
```diff
@@ -25,8 +26,11 @@ def step(filenames: List[str]) -> None:
     :param filenames: Path(s) of the Python source code files.
     """
     project = get_project()
-    assert project
-    config_path = project.get_config_filename()
+
+    if project is None:
+        info("no project linked - assuming files are in current directory.")
```
Comment (akx) [inline valohai_cli/commands/yaml/step.py:31]: Can you try if this works with just

```suggestion
    if project is None:
        info("no project linked - assuming files are in current directory.")
        project = Project(data={}, directory=os.getcwd())
```

i.e. you simulate a project that would live in this directory?

That way you don't need to change everything to accept no Project.

**Review by akx — CHANGES_REQUESTED** [review]

**Review by akx — APPROVED** [review]
Nice and clean 😁

**Pull request merged**